test/lock_activate_test.rb

```diff
@@ -102,14 +106,16 @@ def test_ignore_gems_excluded_by_gemfile
     rack (2.0.3)
     rack-test (0.6.3)
       rack (>= 1.0)
+    hoe (3.0.0)
+    atomic (1.1.16)
```

Comment:
I'm pretty sure these should be in alphabetical order... is this incidentally demonstrating a bug?